tests/test_future.py

```diff
@@ -51,7 +51,7 @@ def test_state(self):
             self.assertTrue(fut.running())
             self.assertFalse(fut.done())
 
-            time.sleep(1.5)
+            fut.result()
```

Comment:
1.5 secs was too short for the cluster and client to fully initialize on slow machines.